src/host/ft_host/API_CursorTests.cpp

```diff
@@ -154,6 +154,6 @@ void CursorTests::TestSetConsoleCursorPosition()
     // Try several cases
     TestSetConsoleCursorPositionImpl(0, 0, TRUE); // Top left corner of buffer
     TestSetConsoleCursorPositionImpl(sbiInitial.dwSize.X - 1, sbiInitial.dwSize.Y - 1, TRUE); // Bottom right corner of buffer
-    TestSetConsoleCursorPositionImpl(sbiInitial.dwSize.X, sbiInitial.dwSize.Y, FALSE); // 1 beyond bottom right corner (the size is 1 larger than the array indicies)

```

Comment:
Man I really need to learn how to spell `indices`.

Reply:
Fwiw, it is a commonly corrupted word. (Although I haven't seen it much recently, it's definitely something I correct often enough.)